site/layouts/shortcodes/card.html

```diff
@@ -22,9 +22,11 @@
 {{- $rtl := .Get "rtl" | default false -}}
 
 <div class="card{{ if eq $borders false }} border-0{{ end }}{{ if and $borders (eq $background "dark") }} border-dark{{ end }}">
-  <svg class="bd-placeholder-img card-img-top" width="100%" height="162" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}" preserveAspectRatio="xMidYMid slice" focusable="false">
+  <svg class="bd-placeholder-img card-img-top" width="100%" height="169" xmlns="http://www.w3.org/2000/svg" role="img" aria-label="{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}" preserveAspectRatio="xMidYMid slice" focusable="false">
     <rect width="100%" height="100%" fill="{{ (index $.Site.Data.grays 3).hex }}"/>
-    <text x="50%" y="50%" fill="{{ (index $.Site.Data.grays 5).hex }}" dy=".3em">{{ if $rtl }}تعليق على الصورة{{ else }}Image cap{{ end }}</text>
+    <svg x="30%" y="30%" width="40%" height="40%" viewBox="0 0 24 24" fill="#00000088" aria-hidden="true" focusable="false">
```

Comment:
Are you sure about the color here? Seems way lighter in https://system.design.orange.com/0c1af118d/p/82eaf1-cards/b/69655e/i/66611055

Reply:
Should be better with the last commit.